csrc/scheduler/normalization_utils.cpp

```diff
@@ -625,5 +625,54 @@ int64_t partialReductionBufferSize(
   return partial_reduction_buffer_size;
 }
 
+int64_t getInnerOuterPersistentBufferBatches(
+    const int64_t inner_vect,
+    const int64_t inner_dim_numel,
+    const int64_t outer_dim_numel,
+    const int64_t warpSize) {
+  // if inner_dim_numel <= 1024, we are doing multiple reductions per block
+  // with a constant batch size of 1
```

Comment:
Please also mention something like: "See also Step 5 of innerOuterPersistentHeuristic"